src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs

```diff
@@ -119,16 +119,16 @@ private string FormatCompletionStatusDescription(int? completionStatus)
         {
             switch (completionStatus)
             {
-                case 1:
+                case (int)CompletionStatus.Continuing:
```

Comment:
This is definitely a use of it - but I was thinking about the original bug that you cannot use a withdrawn from the UI but could from the API - so wouldn't there be some logic which prevent this in the UI - and is that using magic numbers?